Misc/NEWS.d/next/Library/2024-05-25-06-19-46.gh-issue-119535.RXmed8.rst

```diff
@@ -0,0 +1 @@
+On Python 3.14, :mod:`venv` creates an additional ``pythonπ`` symlink on non-Windows systems
```

Comment:
```suggestion
On Python 3.14, :mod:`venv` creates an additional ``pythonπ`` symlink on non-Windows systems.
```

This will be my contribution to this easter egg! (I'm still not really in favor to it but if real-world users are not happy with it, we'll get feedback during the alpha release).

Reply:
There's the argument to not have a NEWS entry for an easter egg...

Reply:
If we do keep the NEWS entry, it should also use the mathematical symbol:

```suggestion
On Python 3.14, :mod:`venv` creates an additional ``python𝜋`` symlink on non-Windows systems.
```

One advantage of mentioning it is that folks just reading the release notes can get a chuckle out of it, even if they never notice the extra symlink themselves.